Main.py

```diff
@@ -0,0 +1,94 @@
+import pygame, sys, time, math, random
+from pygame.locals import *
+
+pygame.init()#Initiates Pygame
+img1 = pygame.image.load('BigSmoke.jpg')  #Imports BigSmoke
```

Comment:
Instead of numbering your variables (a poor practice), you could use an array. Review this [Introduction to Python Lists](http://effbot.org/zone/python-list.htm) for support.